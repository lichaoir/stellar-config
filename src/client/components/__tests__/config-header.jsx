const React = require('react');
const renderer = require('react-test-renderer');
const Header = require('../config-header.jsx');

test('component config-header can be rendered', () => {
  const props = {
    configType: 'source',
    configName: 'vehicles',
  };

  const component = renderer.create(<Header {...props} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
