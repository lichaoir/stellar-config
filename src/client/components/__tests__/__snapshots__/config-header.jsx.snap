// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component config-header can be rendered 1`] = `
<div
  className="tags has-addons"
>
  <span
    className="tag is-medium is-primary"
  >
    source
  </span>
  <span
    className="tag is-medium"
  >
    vehicles
  </span>
</div>
`;
